remoteappmanager/rest/rest_handler.py

```diff
@@ -25,6 +26,44 @@ def get_resource_handler_or_404(self, collection_name):
         except KeyError:
             raise web.HTTPError(httpstatus.NOT_FOUND)
 
+    def write_error(self, status_code, **kwargs):
+        """Provides appropriate payload to the response in case of error.
+        """
+        exc_info = kwargs.get("exc_info")
+
+        if exc_info is None:
+            self.clear_header('Content-Type')
+            self.finish()
```

Comment:
finish with the `status_code` (the argument) as content.  `exc_info` may not be available if `web.RequestHandler.send_error` is manually called.